### PR TITLE
Update road map content for Q3

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR updates the roadmap to reflect the work we’re doing now and next.

### CODE REVIEW

File: app/templates/views/roadmap.html [hunk 1]
```diff
@@ -26,49 +26,77 @@ <h1 class="heading-large">Roadmap</h1>
...
+    <li>Let services upload their own logo for emails and letters</li>
+    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">replace GOV.UK Paas</a> by migrating our database to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
+    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
+    <li>Let services choose their own two-factor authentication method</li>
+    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.features_sms') }}#receive">receive text messages</a> without having to contact the Notify team first</li>
+    <li>Make it easier to find information and guidance on the Notify website</li>
+    <li>Design reusable content patterns for text messages</li>
+    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System that we use</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:40]:
```suggestion
    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
```

File: app/templates/views/roadmap.html [hunk 2]
```diff
@@ -26,49 +26,77 @@ <h1 class="heading-large">Roadmap</h1>
...
+    
+    <li>Add multilingual letter templates so services can provide information in several languages</li>
+    <li>Test reusable content patterns for text messages</li>
+    <li>Review our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">email formatting options</a><li>
+    <li>Get ready to migrate applications from GOV.UK PaaS to Amazon cloud platform</li>
+
+</ul>
+    
     <li>Let services upload their own logo for emails and letters</li>
     <li>Design reusable content patterns for text messages</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:59]:
```suggestion
```

File: app/templates/views/roadmap.html [hunk 3]
```diff
@@ -26,49 +26,73 @@ <h1 class="heading-large">Roadmap</h1>
...
+  <p class="govuk-body">To protect files sent by email, you can now:</p>
 
+  <ul class="govuk-list govuk-list--bullet">
+    
+    <li>ask recipients to enter their email address before they can download a file</li>
+    <li>choose the length of time that a file is available to download</li>
+    
   </ul>
 
+  <h3 class="heading-small">Shown how content desing can reduce the cost of text messages</h3>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:94]:
```suggestion
  <h3 class="heading-small">Shown how content design can reduce the cost of text messages</h3>
```

File: app/templates/views/roadmap.html [hunk 4]
```diff
@@ -26,49 +26,73 @@ <h1 class="heading-large">Roadmap</h1>
...
-<h3 class="heading-small" id="now">Now</h3>
+  <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on now</h2>
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Make it easier for services to pick their own <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.branding_and_customisation')}}">email branding</a></li>
-    <li>Let services preview their branding before sending an email</li>
-    <li>Make <a class="govuk-link govuk-link--no-visited-state" href="/features/email#send-files">sending files by email</a> more secure</li>
+    <li>Let services upload their own logo for emails and letters</li>
+    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">replace GOV.UK Paas</a> by migrating our database to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```
Comment (idavidmcdonald) [inline app/templates/views/roadmap.html:34]: There are a few things technically wrong about this sentence I'm afraid.

We are doing two things right now

Start to migrate our database from one currently hosted in the GOV.UK PaaS to one hosted hosted using AWS RDS
Migrating one of our applications that is currently hosted in the GOV.UK PaaS to being hosted using AWS ECS

You might be able to use those two sentences as is (plus making them link to the relevant pages) or happy to look at a suggestion

Comment (karlchillmaid) [reply app/templates/views/roadmap.html:34]: @idavidmcdonald how’s this?

```suggestion
    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/?nc2=h_ql_prod_fs_rds">Amazon RDS</a></li>
    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

File: app/templates/views/roadmap.html [hunk 5]
```diff
@@ -26,49 +26,73 @@ <h1 class="heading-large">Roadmap</h1>
...
   <ul class="govuk-list govuk-list--bullet">
-    <li>Let services upload their own logo for emails and letters</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Research how we might host Notify ourselves when it’s time to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">replace GOV.UK Paas</a></li>
-  </ul>
+    
+    <li>Add multilingual letter templates so services can provide information in several languages</li>
+    <li>Test reusable content patterns for text messages</li>
+    <li>Review our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">email formatting options</a><li>
+    <li>Get ready to migrate applications from GOV.UK PaaS to Amazon cloud platform</li>
```
[the remark this answers is not in the document]
Comment (karlchillmaid) [reply app/templates/views/roadmap.html:54]:
```suggestion
    <li><li>Migrate a second application from GOV.UK PaaS to Amazon ECS</li>
```

File: app/templates/views/roadmap.html [hunk 6]
```diff
@@ -16,8 +16,8 @@ <h1 class="heading-large">Roadmap</h1>
 
   {{ content_metadata(
     data={
-      "Last updated": "12 July 2022",
-      "Next review due": "11 October 2022"
+      "Last updated": "27 October 2022",
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:19]:
```suggestion
      "Last updated": "28 October 2022",
```

File: app/templates/views/roadmap.html (hunk 4 above)
Comment (idavidmcdonald) [reply app/templates/views/roadmap.html:34]: All good but you can make the AWS link slightly simpler, it only needs point at `https://aws.amazon.com/rds/`, you've got some query string on the end of yours which isn't necessary.

File: app/templates/views/roadmap.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (idavidmcdonald) [reply app/templates/views/roadmap.html:54]: Looks good

File: app/templates/views/roadmap.html [hunk 7]
```diff
@@ -26,49 +26,74 @@ <h1 class="heading-large">Roadmap</h1>
...
   <ul class="govuk-list govuk-list--bullet">
-    <li>Let services upload their own logo for emails and letters</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Research how we might host Notify ourselves when it’s time to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">replace GOV.UK Paas</a></li>
-  </ul>
+    
+    <li>Add multilingual letter templates so services can provide information in several languages</li>
+    <li>Test reusable content patterns for text messages</li>
+    <li>Review our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">email formatting options</a><li>
+    <li><li>Migrate a second application from GOV.UK PaaS to Amazon ECS</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:55]:
```suggestion
    <li>Migrate a second application from GOV.UK PaaS to Amazon ECS</li>
```

File: app/templates/views/roadmap.html [hunk 8]
```diff
@@ -26,49 +26,74 @@ <h1 class="heading-large">Roadmap</h1>
...
 
-    <li>Make it easier for organisations to find the usage data they need</li>
-    <li>Explore ways to share common content patterns</li>
-    <li>Meet our targets for <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">accessibility</a></li>
-    <li>Add multilingual letter templates so services can provide information in several languages</li>
-    <li>Make it easier to start using Notify</li>
+    <li>Improve formatting options for letters</li>
+    <li>Provide more detailed reporting and usage data</li>
+    <li>Explore ways to let services send one-off messages</li>
+    <li>Scale to support services that send a high volume of messages</li>
```
Comment (idavidmcdonald) [inline app/templates/views/roadmap.html:67]: One you might want to run past Irina, but could we remove this one about scaling. I'm not aware of any upcoming scaling work and this implies that we can't currently support services that send a high volume of messages (which users will struggle to identify if they are high volume or not).

Happy to add something more specific back in in the future if we need to

**Review by idavidmcdonald — APPROVED** [review]
Looks good. I left one small comment about a line to hopefully remove.

I haven't got my new laptop up and running fully yet so I haven't tested this locally. When this gets merged, could I ask you to take a look on preview at it to make sure you are happy before letting someone continue deploying it out to staging and production Karl. Just want to make sure we don't have any slight html issues or links not working.

File: app/templates/views/roadmap.html (hunk 8 above)
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:67]:
```suggestion
```

**Pull request merged**